src/libusb/messenger-libusb.cpp

```diff
@@ -0,0 +1,71 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2015 Intel Corporation. All Rights Reserved.
+
+#include "messenger-libusb.h"
+#include "device-libusb.h"
+#include "usb/usb-enumerator.h"
+#include "hw-monitor.h"
+#include "endpoint-libusb.h"
+#include "interface-libusb.h"
+#include "libuvc/uvc_types.h"
+#include "handle-libusb.h"
+
+#include <string>
+#include <regex>
+#include <sstream>
+#include <mutex>
+
+#define INTERRUPT_BUFFER_SIZE 1024
+
+namespace librealsense
+{
+    namespace platform
+    {
+        usb_messenger_libusb::usb_messenger_libusb(const std::shared_ptr<usb_device_libusb>& device)
+            : _device(device)
+        {
+            
+        }
+
+        usb_messenger_libusb::~usb_messenger_libusb()
+        {
+
+        }
+
+        bool usb_messenger_libusb::reset_endpoint(std::shared_ptr<usb_endpoint> endpoint)
+        {
+            int ep = endpoint->get_address();
+            bool rv = control_transfer(0x02, //UVC_FEATURE,
+                                       0x01, //CLEAR_FEATURE
+                                       0, ep, NULL, 0, 10) == UVC_SUCCESS;
+            if(rv)
+                LOG_DEBUG("USB pipe " << ep << " reset successfully");
+            else
+                LOG_DEBUG("Failed to reset the USB pipe " << ep);
```

Comment:
The actual result of the transfer is not registered (reduced to bool) will make it harder for debugging and maintenance

Reply:
usb status added to the log